src/app.html

```diff
@@ -29,6 +29,8 @@
 	.noscroll {
 		overflow: hidden;
 		position: relative;
+		touch-action: none;
+		-ms-touch-action: none;
```

Comment:
If you add this to the app.css file it will autoprefix for you.

Reply:
Smart!

I am still having problems with the background scrolling when the modal is active on iOS Safari.